### ISSUE TITLE
add roles on views

### ISSUE THREAD

**Issue by paulandrieux** [Victoire/victoire#559]
It should be possible to restrict views accessibility by custom permission. These permissions could be handled by custom voters that take the requested view as an argument.

With such a system, we could delete the "author_restricted" field
